### PR TITLE
Pt 167194534 multiple responders per port uw experiments nonces

### PR DESCRIPTION
**Pull request by uwiger**
This replaces PR #2572, adapting the test case to the complexities of GA, and making test case errors a bit easier to understand.

### CODE REVIEW

File: apps/aechannel/test/aesc_fsm_SUITE.erl [hunk 1]
```diff
@@ -170,6 +172,8 @@ groups() ->
     ].
 
 ga_sequence() ->
+      %[ multiple_responder_keys_per_port].
```
Comment (lucafavatella) [inline apps/aechannel/test/aesc_fsm_SUITE.erl]: ?

File: apps/aechannel/test/aesc_fsm_SUITE.erl [hunk 2]
```diff
@@ -356,6 +362,58 @@ stop_node(N, Config) ->
...
+                        ChannelCfg0
+                end,
+            C = create_multi_channel(ChannelCfg, #{mine_blocks => {ask, Me},
+                                                               debug => Debug}, false),
+            [{C, Tx}] = collect_acks_w_payload([C], mine_blocks, 1),
+            TxHash = aeser_api_encoder:encode(tx_hash, aetx_sign:hash(Tx)),
+            mine_blocks_until_txs_on_chain(dev1, [TxHash]),
+            C
+        end,
+    %Cs = [CreateMultiChannel(N, CfgX) || {N, CfgX} <- [{1, Cfg}, {2, Cfg2}]],
```
Comment (lucafavatella) [inline apps/aechannel/test/aesc_fsm_SUITE.erl]: ?

File: apps/aechannel/test/aesc_fsm_SUITE.erl [hunk 3]
```diff
@@ -356,6 +362,58 @@ stop_node(N, Config) ->
 create_channel(Cfg) ->
     with_trace(fun t_create_channel_/1, Cfg, "create_channel").
 
+multiple_responder_keys_per_port(Cfg) ->
+    Slogan = ?SLOGAN,
+    %% Debug = get_debug(Cfg),
+    Debug = true,
```
Comment (lucafavatella) [inline apps/aechannel/test/aesc_fsm_SUITE.erl]: This was maybe meant to rather be deleted, and the line above uncommented?

Comment (uwiger) [reply apps/aechannel/test/aesc_fsm_SUITE.erl]: Yes, though it might be good to keep the debug on for now. :)

**Pull request merged**